modules/ROOT/attachments/admin-api.yaml

```diff
@@ -3972,6 +4093,32 @@ components:
           description: node id
           format: int64
       description: Replica assignment
+    cluster_partition:
+      type: object
+      properties:
+        leader_id:
```

Comment:
It turns out we don't actually return leader_id here. My mistake - forgot to delete it from the schema.